examples/circuit-relaying/README.md

```diff
@@ -0,0 +1,247 @@
+        "Addresses": [
+                "/ip4/127.0.0.1/tcp/4001/ipfs/QmY73BLYav2gYc9PCEnjQqbfSGiqFv3aMsRXNyKFGtUoGF",
+                "/ip4/192.168.1.132/tcp/4001/ipfs/QmY73BLYav2gYc9PCEnjQqbfSGiqFv3aMsRXNyKFGtUoGF",
+                "/ip6/::1/tcp/4001/ipfs/QmY73BLYav2gYc9PCEnjQqbfSGiqFv3aMsRXNyKFGtUoGF",
+                "/ip4/186.4.18.182/tcp/13285/ipfs/QmY73BLYav2gYc9PCEnjQqbfSGiqFv3aMsRXNyKFGtUoGF",
+                "/ip4/186.4.18.182/tcp/13285/ipfs/QmY73BLYav2gYc9PCEnjQqbfSGiqFv3aMsRXNyKFGtUoGF"
+        ],
+        "AgentVersion": "go-ipfs/0.4.14-dev/cb5bb7dd8",
+        "ProtocolVersion": "ipfs/0.1.0"
+}
+```
+
+We can then grab the resolved multiaddr from the `Addresses` array - `/ip4/127.0.0.1/tcp/4004/ws/ipfs/Qm...`. Lets note it down somewhere and move to the next step.
+
+> js ipfs
+
+```
+$ jsipfs daemon
+Initializing daemon...
+Swarm listening on /p2p-circuit/ipfs/QmfQj8YwDdy1uP2DpZBa7k38rSGPvhHiC52cdAGWBqoVpq
+Swarm listening on /p2p-circuit/ip4/0.0.0.0/tcp/4002/ipfs/QmfQj8YwDdy1uP2DpZBa7k38rSGPvhHiC52cdAGWBqoVpq
+Swarm listening on /p2p-circuit/ip4/127.0.0.1/tcp/4003/ws/ipfs/QmfQj8YwDdy1uP2DpZBa7k38rSGPvhHiC52cdAGWBqoVpq
+Swarm listening on /ip4/127.0.0.1/tcp/4003/ws/ipfs/QmfQj8YwDdy1uP2DpZBa7k38rSGPvhHiC52cdAGWBqoVpq
+Swarm listening on /ip4/127.0.0.1/tcp/4002/ipfs/QmfQj8YwDdy1uP2DpZBa7k38rSGPvhHiC52cdAGWBqoVpq
+Swarm listening on /ip4/192.168.1.132/tcp/4002/ipfs/QmfQj8YwDdy1uP2DpZBa7k38rSGPvhHiC52cdAGWBqoVpq
+API is listening on: /ip4/127.0.0.1/tcp/5002
+Gateway (readonly) is listening on: /ip4/127.0.0.1/tcp/9090
+Daemon is ready
+```
+
+Look out for an address similar to `/ip4/127.0.0.1/tcp/4003/ws/ipfs/Qm...` note it down somewhere, and lets move on to the next step. 
+
+### Configure and run the bundled example
+
+Now that we have ipfs installed and initialized, lets set up the included example. This is a standard npm package, so the usual `npm install` should get us going.
+
+```
+npm install
+```
+
+After it finishes, we should be able to run the project with `npm start` and get a similar output to the bellow one:
+
+```
+npm run start
+Server running at http://localhost:1234
+```
+
+The bundled example is a simple chat app that uses another cool ipfs feature - `pubsub`. Lets open up a browser and paste the above url into the address bar. We should see something similar to the following image:
+
+![](./img/img1.png)
+
+### Connect the two browser nodes to the circuit relay
+
+In order for our browser nodes to be able to send messages to each other, we need to first get them connected, but for that we need to use the relay, remember, browser nodes can't be dialed directly because of lack of socket support, so the relay is here to solve that.
+
+Enter the `/ip4/127.0.0.1/tcp/4003/ws/ipfs/...` address noted above into the `Connect to Peer` field and hit the connect button:
```

Comment:
After reading, I realize I don't know *how* circuit relay works. This section might be a good place to mention how these nodes are connecting.

Reply:
> After reading, I realize I don't know how circuit relay works

Thats a very good point - although I'm not sure what is a reasonable amount of info to include here? There is the circuit [readme](https://github.com/libp2p/js-libp2p-circuit), should it be linked somewhere perhaps?

> This section might be a good place to mention how these nodes are connecting.

In this case `Connect to Peer` here, it will use the websockets transport to dial the relay node - which is your usual `ipfs swarm connect <QmPeer...>` operation. 

I'll add that in there so its clear what's going on 👍!